aten/src/ATen/native/cpu/FlashAttentionKernel.cpp

```diff
@@ -38,6 +38,7 @@ inline void _scale_attn_mask_fusion_kernel(
   constexpr int64_t T2_n = 1;
   auto vec_scale = at::vec::VectorizedN<T1, T1_n>(val);
   int64_t i = 0;
+ #if !defined(__powerpc__)
```

Comment:
won't think make this code silently wrong though?

Reply:
This is temporary fix.

Reply:
Could you explain why you think this PR will make it `silently wrong`?

Looks like the change will make PowerPC use only scalar operations and skip the vector operations.  The end result is the PPC will just be slower for this function but still correct.

Reply:
Ho sorry, I didn't realized the loop below continues on the same i and not only does the tail. Sorry about that!